system/jlib/jutil.cpp

```diff
@@ -2417,11 +2420,31 @@ jlib_decl bool querySecuritySettings(bool *          _useSSL,
                 configFileSpec.set(CONFIG_DIR).append(PATHSEPSTR).append("environment.conf");
 #endif
                 Owned<IProperties> conf = createProperties(configFileSpec.str(), true);
-                useSSL = conf->getPropBool("dfsUseSSL", false);
+                StringAttr sslMethod;
+                sslMethod.set(conf->queryProp("dfsUseSSL"));
+                if (sslMethod)
+                {
+                    // checking for true | false for backward compatibility
+                    if ( strieq(sslMethod.str(), "SSLOnly") || strieq(sslMethod.str(), "true") )
```

Comment:
Before last update to this PR, this was:
```
                    if ( strieq(sslMethod.str(), "SSLBoth") || strieq(sslMethod.str(), "true") )
```

Now it's SSLOnly.

For clarification, "true", for old behaviour should be SSLOnly, correct?